tox.ini

```
@@ -1,5 +1,8 @@
 [tox]
 envlist = cov-erase, flake8, {pypy27,pypy36,pypy37,py27,py35,py36,py37}-{tests,flake8,check-manifest,check-newsfragment}, cov-report
+requires=
+    virtualenv>=20.0.35
+    tox-wheel>=0.5.0
```

```suggestion
    tox-wheel>=0.6.0
isolated_build=true
```

https://tox.readthedocs.io/en/latest/config.html#conf-isolated_build
https://github.com/ionelmc/tox-wheel/pull/6